.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+This PR...
+
+## Changes
+
+-
+
+## Screenshots
+
+(prefer animated gif)
+
+## Checklist
+
+- [ ] your PR is pointed at `next` (unless it's a new release)
```

Comment:
is this necessary? if this is our default branch I don't see anyone changing it. For new releases, I don't think we need to require a PR... it should probably just merge next -> master.

Reply:
I would argue yes. `master` is the default. New work goes into `next`. This way, if anyone goes to the project, they see `master` which is the code equivalent to our latest package release.

> I don't think we need to require a PR... it should probably just merge next -> master

Hmm...I disagree here - I think the PR allows for an extra set of eyes. But if you don't, then we can skip that.

Reply:
>  This way, if anyone goes to the project, they see master
I'd argue that it's very often the case that in traditional OSS flow, master is the "yet to be released" work and it's fairly common to not have master be fully up-to-date with the released version.

![image](https://user-images.githubusercontent.com/14339/68615083-540a8a00-0490-11ea-9c77-1d2e4834f50b.png)

![image](https://user-images.githubusercontent.com/14339/68615172-8b793680-0490-11ea-93a3-de1a5caf7a89.png)

![image](https://user-images.githubusercontent.com/14339/68615289-c4b1a680-0490-11ea-869e-cf546c308949.png)

> Hmm...I disagree here - I think the PR allows for an extra set of eyes. But if you don't, then we can skip that.

The reason I don't think this is required is we review every PR that goes in. The merge / release step should be automated via tooling, so there should not be additional code that is added manually.

Reply:
Ah, I see what you're saying. Okay well by default `semnatic-release` creates a new release when we merge into `master`. I set it up the way it is now, to avoid releasing every time we merge to `master`.

So the new question is, how do you want to set it up so we PR into `master` and create a new trigger to generate a release?